ElectronClient/gui/NoteText.jsx

```diff
@@ -1180,6 +1180,13 @@ class NoteTextComponent extends React.Component {
 			};
 		}
 
+		if (command.name === 'focusElement' && command.target === 'noteTextViewer') {
+			fn = () => {
+				if (!this.webviewRef_) return;
+				this.webviewRef_.current.wrappedInstance.webviewRef_.current.focus();
```

Comment:
Please add a `focus()` method on the webview component to avoid this very long chain of properties. Also you need to check if `current` is defined at each step.

Reply:
That makes more sense, will modify.